### ISSUE TITLE
Many warnings, please upgrade

### ISSUE THREAD

**Issue by ppKrauss** [mkdocs/mkdocs-basic-theme#5]
After copy folder `basic_theme` to my project and run with `mkdocs build` , many WARNINGs

```
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/user/gits/miniguias/DiscussOKBr/site 
WARNING -  Template variable warning: 'google_analytics' is being deprecated and will not be available in a future version. Use 'config.google_analytics' instead. 
WARNING -  Template variable warning: 'favicon' is being deprecated and will not be available in a future version. Use '{{ base_url }}/img/favicon.ico' instead. 
WARNING -  Template variable warning: 'previous_page' is being deprecated and will not be available in a future version. Use 'page.previous_page' instead. 
WARNING -  Template variable warning: 'site_name' is being deprecated and will not be available in a future version. Use 'config.site_name' instead. 
WARNING -  Template variable warning: 'next_page' is being deprecated and will not be available in a future version. Use 'page.next_page' instead. 
WARNING -  Template variable warning: 'toc' is being deprecated and will not be available in a future version. Use 'page.toc' instead. 
WARNING -  Template variable warning: 'page_title' is being deprecated and will not be available in a future version. Use 'page.title' instead. 
WARNING -  Template variable warning: 'repo_url' is being deprecated and will not be available in a future version. Use 'config.repo_url' instead. 
WARNING -  Template variable warning: 'repo_name' is being deprecated and will not be available in a future version. Use 'config.repo_name' instead. 
WARNING -  Your theme does not appear to contain a 'main.html' template. The 'base.html' template was used instead, which is deprecated. Update your theme so that the primary entry point is 'main.html'. 
WARNING -  Template variable warning: 'content' is being deprecated and will not be available in a future version. Use 'page.content' instead. 
```

**Comment by d0ugal** [comment]
Thanks for the report - this example hasn't yet been updated to reflect the changes in MkDocs. It is a fairly simple search and replace as the warning suggestions.

**Comment by shielamp13** [comment]
#3

**Comment by waylan** [comment]
Any remaining issues were fixed in #11.
